File: saude_java/app/src/main/java/saude_java/Aluno.java
package saude_java;

public class Aluno {

    private double peso;
    private double altura;
    private int idade;
    private String sexo;
    private double cintura;
    private double quadril;

    public Aluno(double peso, double altura, int idade, String sexo, double cintura, double quadril) {
        this.peso = peso;
        this.altura = altura;
        this.idade = idade;
        this.sexo = sexo;
        this.cintura = cintura;
        this.quadril = quadril;
    }

    public double imc() {
        return IMC.calcular(peso, altura);
    }

    public double tmb() {
        // Altura convertida para centímetros
        return TMB.calcular(peso, altura * 100, idade, sexo);
    }

    public double rcq() {
        return RCQ.calcular(cintura, quadril);
    }

    public double pg() {
        return PG.calcular(imc(), idade, sexo);
    }

    // Getters para as variáveis (se necessário)
    public double getPeso() {
        return peso;
    }

    public double getAltura() {
        return altura;
    }

    public int getIdade() {
        return idade;
    }

    public String getSexo() {
        return sexo;
    }

    public double getCintura() {
        return cintura;
    }

    public double getQuadril() {
        return quadril;
    }
}

File: saude_python/src/saude_python/aluno.py
from src.saude_python.imc import IMC
from src.saude_python.tmb import TMB
from src.saude_python.rcq import RCQ
from src.saude_python.pg import PG

class Aluno:
    def __init__(self, peso=None, altura=None, idade=None, sexo=None, cintura=None, quadril=None):
        self.peso = peso
        self.altura = altura
        self.idade = idade
        self.sexo = sexo
        self.cintura = cintura
        self.quadril = quadril

    def imc(self):
        return IMC.calcular(self.peso, self.altura)

    def tmb(self):
        # Multiplicamos a altura por 100 para converter para centímetros
        return TMB.calcular(self.peso, self.altura * 100, self.idade, self.sexo)

    def rcq(self):
        return RCQ.calcular(self.cintura, self.quadril)

    def pg(self):
        return PG.calcular(self.imc(), self.idade, self.sexo)
